src/MarkdownFileCompletionItemProvider.ts

```diff
@@ -45,11 +40,6 @@ export class MarkdownFileCompletionItemProvider implements vscode.CompletionItem
           }
           return item;
         });
-        return items;
-        break;
-      default:
-        return [];
```

Comment:
I think we should probably leave the default case returning empty array, no? Is there any reason not too?

Reply:
Ummm, a JS/TS function will return null if there's no return statements executed. So I think "only return some results when we actually have some results" feels more natural IMHO. But feel free if you want the default case back, you're the lord here :)

Reply:
Yea, I am just wondering if anything downstream expects `[]` vs `undefined` so gonna add a default []

https://github.com/kortina/vscode-markdown-notes/pull/76